base_jsonify/readme/ROADMAP.rst

```diff
@@ -0,0 +1 @@
+Make a distinction between float 0.0L and no value
```

Comment:
@PierrickBrun IMO this issue has been solved by 3ca15ceada6c48ac9e6a87f41c41719dcfe4ca89

Reply:
indeed